composer/utils/collect_env.py

```diff
@@ -104,7 +104,15 @@ class ComposerEnv(NamedTuple):
 def get_composer_commit_hash() -> Optional[str]:
     # Use PEP-610 to get the commit hash
     # See https://packaging.python.org/en/latest/specifications/direct-url/
-    files = importlib_metadata.files('mosaicml')
+    # try both package names that composer is released under
+    try:
+        files = importlib_metadata.files('mosaicml')
+    except importlib_metadata.PackageNotFoundError:
+        try:
+            files = importlib_metadata.files('composer')
+        except importlib_metadata.PackageNotFoundError:
+            return
```

Comment:
```suggestion
    # Try both package names that Composer is released under
    try:
        files = importlib_metadata.files('mosaicml')
    except importlib_metadata.PackageNotFoundError:
        try:
            files = importlib_metadata.files('composer')
        except importlib_metadata.PackageNotFoundError:
            return
```
```suggestion
    try:
        files = importlib_metadata.files('mosaicml')
    except importlib_metadata.PackageNotFoundError:
        try:
            files = importlib_metadata.files('composer')
        except importlib_metadata.PackageNotFoundError:
            return
```
Nit: I think we try to capitalize start of comments